kickstart/update-simulator.yaml

```diff
@@ -1,6 +1,6 @@
 summary: Update the simulator for the $SRYYYY game
 
-priority: major
+priority: should
```

Comment:
This is complex. If we're having a virtual competition, IMO the simulator being updated is a "must". If it's just a dev aid, then I agree with "should".